.circleci/config.yml

```diff
@@ -1,20 +1,136 @@
 version: 2
+
+# Settings common to each job.
+anchor_1: &defaults
+  working_directory: ~/angular-cli
+  docker:
+    - image: angular/ngcontainer
+
+# Restore cache based on package-lock.json checksum for branch.
```

Comment:
in other repos, I removed the branch segment, because are we really worried about cache collisions? And if two branches have the same package-lock.json, we should allow them to share a cache, right?

Reply:
I think that makes sense. Will remove as well.

Reply:
Worth highlighting this section: https://circleci.com/docs/2.0/caching/#caching-strategy-tradeoffs

Specifically, changing node/npm will be problematic with cache without branch. But it's just something to keep in mind really. The tradeoff is that all PRs/branches will always be much faster to startup.